data/json/recipes/other/tool.json

```diff
@@ -134,9 +134,8 @@
     "type": "recipe",
     "category": "CC_OTHER",
     "subcategory": "CSC_OTHER_TOOLS",
-    "skill_used": "fabrication",
-    "difficulty": 4,
-    "skills_required": [ "fabrication", 4 ],
+    "skill_used": "electronics",
+    "difficulty": 3,
```

Comment:
Comparing it with other kitchen appliance, now that you made me aware that they use the electronics skill, I like it. 
The food processor just chop up things raging from chunk-sized to finely grounded. I think is a rather simple machine, and I wanted to make it simple to craft too.